example/example.json

```diff
@@ -99,14 +97,87 @@
             "description": "Invalid resource identifier"
           }
         },
-        "tags": [
-          "Foo With Spaces"
-        ],
+        "tags": ["Foo With Spaces"],
         "summary": "Get foos with spaces",
         "description": " Get foos with spaces",
         "operationId": "getFoosWithSpaces"
       }
     },
+    "/api/v2/foo/{id}": {
+      "patch": {
+        "responses": {
+          "204": {
+            "description": "No content"
+          },
+          "401": {
+            "description": "Invalid access token"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "404": {
+            "description": "Invalid resource identifier"
+          }
+        },
+        "summary": "Update foo",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "Foo id",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "format": "string",
+              "description": "Foo id"
+            }
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "oneOf": [
+                  {
+                    "type": "object",
```

Comment:
I was actually expecting this to be a ref to `example.FooPatchOperation`, since it is registering this type below. I must be missing something when I'm recursively constructing the inner types that they're being included as-is, and not replaced with a ref.

Reply:
Fixed this by pulling the recursion up a level 👍